src/cirrus/environment.py

```diff
@@ -24,12 +34,21 @@ def cirrus_home():
         return os.environ['CIRRUS_HOME']
     
     home = inspect.getsourcefile(cirrus)
-    # from the cirrus init py in the venv, we need to
-    # move up 5 dirs to get the install directory
-    for i in range(6): 
-        home = os.path.dirname(home)
+    if ('venv' in home) and ('site-packages' in home):
+        # we are in a pip installed virtualenv site-packages
+        # from the cirrus init py in the venv, we need to
+        # move up 5 dirs to get the install directory
```

Comment:
Pip install drops it in the venv, eg: venv/lib/python2.7/site-packages 
Thinking about it, having venv in that check is dumb. 
Other path is the old installed via repo or local dev approach